src/data/builder/builder.ts

```diff
@@ -9,9 +9,13 @@ export const buildersData: BuildersDTO = [
       'OKP4 Governance Creator',
       'OKP4 Law-Stone Builder',
       'OKP4 Objectarium Explorer',
-      'OKP4 Young Druid'
+      'OKP4 Young Druid',
+      'OKP4 Dataverse precursor',
+      'OKP4 Prolog query master',
+      'OKP4 interface innovator',
+      'OKP4 visionary contributor'
```

Comment:
@antho31 
The terms above (lines 8-12)  are capitalized, wouldn't it be better to keep this formatting?

Reply:
Yes you're right, better capitalized

Reply:
@houssemmimoun27 can you fix it? :)